src/views/detail/detail.js

```diff
@@ -15,13 +16,21 @@ const component = urlParams.get('component')
 const vm = new Vue({
   el: '#app',
   data: {
-    componentToRender: mock,
+    componentTitle: component.replace('-', ' '),
+    html: '',
+    scss: '',
+    js: null,
     currentList: category
   },
   mounted() {
-    const elem = document.querySelector('#host')
-    const shadowRoot = elem.attachShadow({mode: 'open'})
-    shadowRoot.innerHTML = `${mock.html}<style>${mock.css}</style>`
+    getComponentFiles(category, component).then(response => {
+      this.html = response.html
+      this.scss = response.scss
+      const elem = document.querySelector('#host')
+      const shadowRoot = elem.attachShadow({mode: 'open'})
+      shadowRoot.innerHTML = `${this.html}<style>${this.scss}</style>`
+    })
+    getAuthors(category, component).then(response => console.log(response))
```

Comment:
Is this intentional, or was forgotten here from debugging?

Reply:
Forgotten, my bad 🤦‍♂